src/tests/unit/tests/electron/platform/android/test-configs/automated-checks/results-filter.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { UnifiedResult } from 'common/types/store-data/unified-data-interface';
+import { automatedChecksResultsFilter } from 'electron/platform/android/test-configs/automated-checks/results-filter';
+
+describe('automatedChecksResultsFilter', () => {
+    it('filters results properly', () => {
```

Comment:
It would be nice for the test name to clarify what the actual requirement is; "properly" doesn't tell me at a glance what the expected behavior is